### DISCUSSION TITLE
GPU mode only show GPU fan speed

### DISCUSSION THREAD

**Discussion by ntl2911** [seerge/g-helper#567] [General]
Hello, there are two minors issue for me.
1) The Dashboard is not complete. Selection for GPU mode and screen mode is not there. Photo is attached: 
<img width="293" alt="image" src="https://github.com/seerge/g-helper/assets/135945006/f8d4bdb7-5823-4198-ac49-32c0e8ced7e9">

2) Anime Matrix Clock option is displayed as a horizontal bar, where a small section of the time is cut off (due to the diagonal arrangement) tl;dr: is there any way to make the anime matrix display just like Armoury Crate does?

Thanks a lot for this app, I'm so fed up with Armoury Crate

**Comment by seerge** [comment]
@ntl2911 hello.

1. GPU section is hidden, since 2020 model doesn't support Eco mode, so whole section makes no sense.
2. As for clock layout - it's again done like that only for old 2020 models, as other user's reported that "normal" diagonal layout doesn't fit properly there. Since I don't have that device, it's very difficult for me to test / debug this :)

**Reply by ntl2911** [reply]
thank you for your prompt reply. TBF only when the time hit 10, 11 and 12 o'clock, a small section of the first "1" is cut out.

**Reply by seerge** [reply]
@ntl2911 can you post a photo ?

**Reply by ntl2911** [reply]
so when in AM/PM mode, the length would reach far toward the left (blue circle), x mark is where the letter will be cut off.
![612e415ba72876762f391](https://github.com/seerge/g-helper/assets/135945006/4efe75be-18d7-43e2-bd21-3b8b9add4d1f)

When in 12 hour format, it does not have any problem
![58a47fc699b548eb11a4](https://github.com/seerge/g-helper/assets/135945006/b6eb7ba9-8305-4bcd-8e99-25c7a46b7685)

**Reply by seerge** [reply]
Can you try this build ? 
[GHelper.zip](https://github.com/seerge/g-helper/files/11706082/GHelper.zip)

**Reply by ntl2911** [reply]
I did an auto update yesterday and the size of the clock is fitted nicely, never got a chance to download your suggested build. thanks for the help.
